features.txt

```diff
@@ -880,6 +880,13 @@ tweaks.recipe_book_auto_craft
 	desc:
 		Recipe book crafts items instead of moving them to the crafting area.
 
+tweaks.no_heavy_minecarts
+    name: No Heavy Minecarts
```

Comment:
features.txt must be indented with tabs - I'll fix this, but for future reference.

Reply:
oh weird, idea decided to use four spaces instead of tabs even though the document is all in tabs